networking/onprem-google-access-dns/variables.tf

```diff
@@ -18,16 +18,19 @@ variable "bgp_asn" {
   description = "BGP ASNs."
```

Comment:
I would redesign variables, using a single "regions" variable where key is the region name, values an object with parameters for each region (ASN, IP range, etc.).

Then parameterize main.tf so we create resources for 1-n regions.